portal-bridge/src/types/state/walk_trie_diff.rs

```diff
@@ -0,0 +1,101 @@
+use alloy_primitives::{Bytes, B256};
+use eth_trie::{decode_node, node::Node};
+use hashbrown::HashMap as BrownHashMap;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+pub struct AccountProof {
+    pub path: Vec<u8>,
+    pub proof: Vec<Bytes>,
+}
+
+/// This struct takes in a root hash and a hashmap of changed nodes, then you can call an iterator
```

Comment:
> parent_info

`path_backwards` says it how it is. Not a fan of `parent_info` as it has nothing to do this the parents info, it is a the path to the parent, so I could comprise on `paths_to_parent`

Reply:
It's more path **from** parent.

To me, parent_info makes sense because it's a map where key is a node and value are information about it's parent in the trie: hash+path_from_parent.

I'm fine with just `parent` to simplify and avoid confusion.

Reply:
`parent` is too simple. People should be able to understand what the hashmap stores just from reading the name. They shouldn't need to read a comment.

So I would want it to indicate it is a reverse map from the child to the path from the parent